store/storage/sqlite/db.go

```diff
@@ -175,13 +190,40 @@ func (db *Database) ApplyChangeset(version uint64, cs *store.Changeset) error {
 }
 
 func (db *Database) Prune(version uint64) error {
-	stmt := "DELETE FROM state_storage WHERE version <= ? AND store_key != ?;"
+	tx, err := db.storage.Begin()
```

Comment:
can we get a godoc on how prune works here

Reply:
yes, of course.